Automattic/jetpack
# Social: Ensure the panel is rendered during pre/post publish

## Proposed changes:

The change in #36362 meant that the pre/post publish panels would only be rendered when the sidebar was open. This change ensures they're rendered outside of the JetpackPluginSidebar component.

### Other information:


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Go to the editor with Jetpack active and the Publicize module enabled
* Close the sidebar
* Click Publish
* Without this change there will be no social settings rendered
* With it, the settings should be rendered as expected
